onnxruntime/core/graph/graph_utils.h

```diff
@@ -41,6 +41,9 @@ bool IsGraphInput(const Graph& graph, const NodeArg* input);
 /** Checks if the given node has only constant inputs (initializers). */
 bool AllNodeInputsAreConstant(const Graph& graph, const Node& node);
 
+/** Checks if the given NodeArg is constant, i.e., it appears in the graph's initializers but not in its inputs. */
+bool NodeArgIsConstant(const Graph& graph, const NodeArg& node_arg);
```

Comment:
I would've preferred IsNodeArgConstant as it clearly signifies a question and a bool answer. NodeArgIsConstant sounds like an assertion.